bases/shared/entrypoint.sh

```diff
@@ -1,897 +1,10 @@
 #!/bin/bash
-set -x
-set +e
+set -ex
```

Comment:
These do not mean the same thing.  Do you really want `set -ex`?  That's the same as `set -e` and `set -x`.  `set -e` will silently terminate the script if any commands return a nonzero exit code.  Has this script been audited to ensure that all exit codes are handled carefully to guarantee no unexpected termination?

Reply:
Well I'm curious that why it was set +e at first place. We should stop further execution when any inside command returns non-zero status.
By set +e, we might be supressing the issues directly/indirectly affecting the containers.

Also I think @michaelfig you have raised a good point about verification of exitcode handling and I think "set -ex" might help to identify/fix such problems.

Reply:
If you'd like to take on this work of auditing and handling the exit codes, I recommend doing it in a separate PR after this one lands (using `set +e` in this current PR and only enabling `set -ex` only in the new one).

The main reason I'd like this separated is that the current PR should be a refactor and not a behaviour change.

Reply:
I agree. This was unintentional. Reverted